scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -428,8 +428,12 @@ def dijkstra(csgraph, directed=True, indices=None,
         will decrease computation time by aborting calculations between pairs
         that are separated by a distance > limit. For such pairs, the distance
         will be equal to np.inf (i.e., not connected).
-
-        .. versionadded:: 0.14.0
```

Comment:
still missing the old `versionadded`, which is for another argument -- I think that should just stay where it was

Reply:
ah i see.. fixed